### PR TITLE
Update README for #43, #71

### LINKED ISSUE #57: Update README for #43, #71
In first paragraph of Usage section, clarify that autolinks having three parts - pattern, template and scope.

https://github.com/mattermost/mattermost-plugin-autolink/pull/43 added an optional scope parameter to define which team/channel the defined autolink applies to. 

https://github.com/mattermost/mattermost-plugin-autolink/pull/71 added the /autolink command, and the `WordBoundary`, `Disabled`, and `Name` link attributes.

### PR DESCRIPTION
**Pull request by rohanjulka19**
Fixes #57

### CODE REVIEW

**Comment by levb** [comment]
@rohanjulka19 are you a first-time contributor? If so, welcome, and you should sign the CLA, see the "checks" section below.

**Comment by rohanjulka19** [comment]
@levb I have  signed the CLA

File: README.md [hunk 1]
```diff
@@ -17,15 +17,14 @@ Use it to add custom auto-linking on your Mattermost system, such as adding link
...
 
 2. Modify your `config.json` file to include the types of regexp patterns you wish to match, under the `PluginSettings`. See below for an example of what this should look like.
 
 ## Usage
 
-Autolinks have 2 parts: a **Pattern** which is a regular expression search pattern utilizing the [Golang regexp library](https://golang.org/pkg/regexp/), and a **Template** that gets expanded. You can create variables in the pattern with the syntax `(?P<name>...)` which will then be expanded by the corresponding template.
+Autolinks have 3 parts: a **Pattern** which is a regular expression search pattern utilizing the [Golang regexp library](https://golang.org/pkg/regexp/), a **Template** that gets expanded and an optional **Scope** parameter  to define which team/channel the autolink applies to. You can create variables in the pattern with the syntax `(?P<name>...)` which will then be expanded by the corresponding template.
```
Comment (aaronrothschild) [inline README.md]:
```suggestion
Autolinks have 3 parts: a **Pattern** which is a regular expression search pattern utilizing the [Golang regexp library](https://golang.org/pkg/regexp/), a **Template** that gets expanded and an optional **Scope** parameter to define which team/channel the autolink applies to. You can create variables in the pattern with the syntax `(?P<name>...)` which will then be expanded by the corresponding template.
```

**Comment by hanzei** [comment]
@aaronrothschild Please merge if you think this is fine

**Comment by levb** [comment]
(@aaronrothschild : oops - I thought I committed your extra space removal in-place, but it error-ed out for me, please edit it separately if needed?)

**Pull request merged**